### DISCUSSION TITLE
Message while deploying to Heroku "celery 5.1.0 has a non-standard dependency specifier pytz>dev"

### DISCUSSION THREAD

**Discussion by Dandada1993** [celery/celery#8643] [General]
While deploying project to Heroku the following warning is displayed "DEPRECATION: celery 5.1.0 has a non-standard dependency specifier pytz>dev. pip 24.0 will enforce this behaviour change. A possible replacement is to upgrade to a newer version of celery or contact the author to suggest that they release a version with a conforming dependency specifiers. Discussion can be found at https://github.com/pypa/pip/issues/12063"

I upgrade from Celery 4 to 5 but the message is still there. In the meta data the requirement for pytz is listed as (>dev). Can this be corrected?

**Comment by auvipy** [comment]
just update celery to 5.3.5+

**Reply by Dandada1993** [reply]
Thanks.

**Comment by anshulesh-IIMB** [comment]
pip show celery.
pip3 install --upgrade celery.
pip3 show celery.
